readme.txt

```diff
@@ -0,0 +1,16 @@
+=== WP-Stateless - BuddyBoss Platform Addon ===
+Contributors: usability_dynamics, andypotanin, ideric, maxim.peshkov, Anton Korotkoff, planvova, balexey88
+Donate link: https://udx.io
+Tags: google, google cloud, google cloud storage, cdn, uploads, media, stateless, backup
+License: GPLv2 or later
+Requires PHP: 8.0
+Requires at least: 5.0
+Tested up to: 6.3.1
+Tested with BuddyBoss Platform: 2.4.30
+Stable tag: 0.0.1
+
+Provides compatibility between BuddyBoss Platform WP-Stateless plugins.
```

Comment:
@balexey88 
```
between BuddyBoss Platform WP-Stateless plugins.
```
please add `and` between Plugin names
```
between BuddyBoss Platform and WP-Stateless plugins.
```

Reply:
Thanks, fixed!